### ISSUE TITLE
VMware: VM import fails with default template

### ISSUE THREAD

**Issue by shwstppr** [apache/cloudstack#8432]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Server
~~~

##### CLOUDSTACK VERSION

~~~
4.19RC1
~~~

##### CONFIGURATION

VMware with advanced zone

##### OS / ENVIRONMENT

N/A

##### SUMMARY

VMware VM import fails when importing a VM without specifying a template.

##### STEPS TO REPRODUCE

~~~
1. Deploy a VM and unmanage it
2. Try to import the VM without specifying a template
~~~

##### EXPECTED RESULTS

~~~
VM import returns success
~~~

##### ACTUAL RESULTS

~~~
VM import fails
~~~
![Screenshot from 2024-01-03 14-03-24](https://github.com/apache/cloudstack/assets/153340/9fb40dce-3a7f-4732-b7b7-f6355584ae08)

**Comment by shwstppr** [comment]
Seems to be caused with this change https://github.com/apache/cloudstack/blob/main/server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java#L1114

cc @nvazquez @andrijapanicsb

**Comment by nvazquez** [comment]
Thanks @shwstppr - I have added a fix on #8429

**Comment by shwstppr** [comment]
Fixed with #8429
